### PR TITLE
Milestone 2 Portfolio: Mobile version 

### PR DESCRIPTION
**Pull request by michael-duke**
## Portfolio milestone 2 : Mobile version 

In this milestone I :
- Understood how to parse a Figma design to create a UI.
- Used Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Built a personal portfolio mobile site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
         </a>
       </div>
     </header>
+    <section class="works">
+      <div class="projects">
+        <img src="images/snapshoot-project-1.png" alt="snapshoot-project-1" />
+        <p class="project-title">Tonic</p>
```
Comment (mirouhml) [inline index.html:113]: - [ ] Please know that each project title should be wrapped in a `hx` tag, kindly change that. 😄

File: index.html [hunk 2]
```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
...
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
```
Comment (mirouhml) [inline index.html:124]: - [ ] Please know that the project description should be wrapped in a `p` tag, kindly change that. 🙂

File: styles/projects.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+.projects {
+  display: flex;
+  flex-direction: column;
+  gap: 0.75rem;
+  min-height: 100vh;
+  border: 0.0625rem solid #dfe1e6;
+  border-radius: 1rem;
+  background-color: #fff;
+  padding: 1rem;
+}
```
Comment (mirouhml) [inline styles/projects.css:18]: Please know that each project card is too long, please check the screenshots below, the left one is from your website and the one on the right is the one from the Figma design:

<div>
  
  
</div>

Kindly follow the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to fix this. 😁😁

(Hint: You can remove the `min-height` you set and that should fix it). 😉

File: index.html [hunk 4]
```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
...
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+    </section>
+    <section class="about-section">
+      <div class="top-section">
+        <p class="title">About Myself</p>
```
Comment (mirouhml) [inline index.html:201]: - [ ] Same thing here about the title, it should be wrapped in a `hx` tag, kindly change that. 😄

File: index.html [hunk 5]
```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
...
+        <p class="outro">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="social-media">
+          <p class="supporting-text">Let's connect</p>
```
Comment (mirouhml) [inline index.html:209]: - [ ] This also should be wrapped in a `hx` tag, kindly change that. 😄

File: index.html [hunk 6]
```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
...
+                  fill="#505F79"
+                />
+              </svg>
+            </a>
+          </div>
```
Comment (mirouhml) [inline index.html:289]: - [ ] Please know that the list of icons should be grouped by a `ul` tag and each `a` should be in an `li` tag, kindly fix that. 😁

File: styles/about.css [hunk 7]
```diff
@@ -0,0 +1,85 @@
+/* About section */
+.about-section {
+  padding: 7.125rem 1.5rem;
+  width: 100vw;
+  border-top-right-radius: 5.0625rem;
+  background-color: #fff;
+}
```
Comment (mirouhml) [inline styles/about.css:7]: - [ ] This section should be aligned using a `flexbox`, please fix that.😁

**Review by mirouhml — CHANGES_REQUESTED** [review]
Hi @michael-duke, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## Highlights
- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (michael-duke) [reply index.html:113]: **Made the title with h2 tag** 👍. Hope that works

File: index.html (hunk 2 above)
Comment (michael-duke) [reply index.html:124]: **Changed all `div`s having the `.project-desc` class to `p` tag the**

File: index.html (hunk 4 above)
Comment (michael-duke) [reply index.html:201]: - Resolved the issue thank your for this feedback @mirouhml

File: index.html (hunk 5 above)
Comment (michael-duke) [reply index.html:209]: Changed it to an `h3` tag

File: index.html (hunk 6 above)
Comment (michael-duke) [reply index.html:289]: - Made the required changes good tip @mirouhml

File: styles/about.css (hunk 7 above)
Comment (michael-duke) [reply styles/about.css:7]: - I have made made the changes , kindly let me know if they went through 😄

Comment (michael-duke) [reply styles/about.css:7]: @mirouhml Thank you for your good tips

Comment (mirouhml) [reply styles/about.css]: @michael-duke Good job on making the changes, please submit again through the dashboard.

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @michael-duke  🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great work on making the changes requested by a previous reviewer 👏
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] I can see you are using the Microverse `MIT.md` template, good job 👏 but you forgot to cutomize your `MIT.md` file, I suggest adding your name and customizing it 👌.
> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by michael-duke** [comment]
**@Bondok6 I will make the optional changes, thank you for the feedback**

**Pull request merged**